contracts/staking/stakeManager/StakeManager.sol

```diff
@@ -645,9 +742,8 @@ contract StakeManager is IStakeManager, StakeManagerStorage {
             ValidatorShare validatorShare = ValidatorShare(
                 validators[validatorId].contractAddress
             );
-            delegationAmount = int256(validatorShare.activeAmount());
             rewards = rewards.add(validatorShare.withdrawRewardsValidator());
-            validatorShare.lock();
+            delegationAmount = int256(validatorShare.lockContract());
```

Comment:
lockContract() returning an int256 sounds bad naming.

Reply:
trying to reduce internal calls. There are some other changes that need lock/unlock, will change naming and design there only. (depending on those requirements)